docs/application/tizen-studio-0/setup/install-sdk.md

```diff
@@ -1,105 +1,150 @@
+.docs-ui-started 
+.docs-ui-ide:before, 
+.docs-ui-star
+.docs-ui-wear:before, 
+.docs-ui-star 
+.docs-ui-t:before, 
+.docs-ui-start 
+.docs-ui-w:before, 
+
+.docs-ui-started .docs-ui-watch:before 
+{
+    height: 85px;
+    margin: 0 auto 25px;
+    align: center;
+    background-position: 0 6px;
+}
+.docs-ui-started .docs-ui-wearable:before {
+    content: "";
+    display: block;
+    margin: auto;
+    position: relative;
+    width: 85px;
+    height: 90px;
+    background: url(./media/win.png) no-repeat center top;
+    background-position: 0 0 !important;
+}
+.docs-ui-started .docs-ui-wearable {
+    width: 33%;
+    padding-left: 50;
+    /* border-right: 1px solid #d1d1d1; */
+}
+.docs-ui-started .docs-ui-tv:before {
+    content: "";
+    margin: auto;
+    position: relative;
+    display: block;
+    width: 85px;
+    height: 90px;
+    background: url(./media/linux1.png) no-repeat center top;
+    background-position: 0 0 !important;
+}
+.docs-ui-started .docs-ui-tv {
+    width: 33%;
+    padding-left: 50;
+}
+
+.docs-ui-started .docs-ui-widget:before {
+    content: "";
+    margin: auto;
+    position: relative;
+	display: block;
+    width: 85px;
+    height: 90px;
+    background: url(./media/apple1.png) no-repeat center top;
+    background-position: 10 0 !important;
+}
+.docs-ui-started .docs-ui-widget {
+    width: 33%;
+    padding-left: 50;
+    /* border-right: 1px solid #d1d1d1; */
+}
+}
+
+
+div {
+    display: block;
+}
+
+ul.a{
+a.docs-btn-more {
+    display: inline-block;
+    font-size: 13px;
+    color: #008aee;
+}}
+</style>
+
+<section id ="main">
+
+This section explains about options available to install Tizen Studio on your development hardware. Tizen Studio runs on all major operating systems like Windows, macOS, Ubuntu. You can install any release of the Tizen Studio. However, it is recommended that you install the most recent release. 
+
+
+
+- Plan your installation: check for the necessary hardware and software requirements. 
+- Install Tizen Studio:  follow the installation guides for each operating system.
+- Configure: After installation, to have optimal development experience, configure various Tizen Studio components.  
+
+Tizen Studio runs on Windows, Ubuntu, as well as macOS and for installation guide, click the following as per target operating system: 
+<br>
+<br>
+<br>
+<div class="docs-ui-started">
+  <div class="docs-ui-wearable">
+    <span>
+    <!--<a href="setup/windows.md" class="docs-btn-more">Windows&reg</a>-->
+	    <a href="windows" class="docs-btn-more">Windows&reg</a>   
+    </span>
+  </div>
+
+  <div class="docs-ui-tv">
+    <span>
+     <!--
+     <a href="/application/tizen-studio/setup/ubuntu.md" class="docs-btn-more">Ubuntu&reg</a><br>-->
+    <a href="ubuntu" class="docs-btn-more">Ubuntu&reg</a><br>
+    </span>
+  </div>
+ 
+   <div class="docs-ui-widget">
+    <span>
+    <!--<a href="/application/tizen-studio/setup/mac.md" class="docs-btn-more">MacOS&reg</a><br>-->
```

Comment:
Same as above.

Reply:
I have made the necessary changes, please review and give your valuable comments.